basicmessage_storage/pyproject.toml

```diff
@@ -43,7 +43,7 @@ ignore = [
     "D203", "D204", "D213", "D215", "D400", "D401", "D404", "D406", "D407",
     "D408", "D409", "D413",
     "D202", # Allow blank line after docstring
-    "D104", # Don't require docstring in public package
+    "D101", "D102", "D104", "D106", "D107",
```

Comment:
applies to all... should we deviate from ACA-Py itself? i think these rules should be the same. even if you put in placeholder comments with TODO and they could get assigned to the creators to fill in properly.

Reply:
Oh. I didn't realize these were the same as aca-py. I'll change it back and update.